### PR TITLE
Updates items to address selenium tests

### PR DESCRIPTION
**Pull request by bretterer**
## Description:
- This PR updates the vuepress-theme-default files and migration script to address failing selenium tests.
- Known issues are blog and quickstart being bypassed
- CSS for mobile page navigation tests failing still.

### CODE REVIEW

File: packages/@okta/migrate-from-jekyll/index.js [hunk 1]
```diff
@@ -116,8 +116,8 @@ function buildFile(file) {
 
 function run() {
   // console.log(`${chalk.bold.green("Migrating from okta.github.io source repo.")}\n`)
-  execSync(`rm -rf okta.github.io`);
-  cloneRepo('https://github.com/okta/okta.github.io.git')
+  // execSync(`rm -rf okta.github.io`);
```
[the remark this answers is not in the document]
Comment (bretterer) [reply packages/@okta/migrate-from-jekyll/index.js]: Yes, just not at the moment.

File: packages/@okta/vuepress-site/data/event-types.json [hunk 2]
```diff
@@ -409,8 +409,8 @@
     "description" : "",
     "tags" : [ ]
   }, {
-    "id" : "callback",
```
[the remark this answers is not in the document]
Comment (bretterer) [reply packages/@okta/vuepress-site/data/event-types.json]: Yes

**Comment by wayneearl-okta** [comment]
Looks good to me

**Comment by wayneearl-okta** [comment]
Looks like the stop scripts are throwing errors. Hmmm...

**Pull request merged**